.github/workflows/main.yml

```diff
@@ -84,13 +92,20 @@ jobs:
         key: testbed-${{ matrix.os }}-${{ hashFiles('testing') }}-5
     - name: Verify or create the testbed
       if: steps.cache-key.outputs.cache-hit != 'true'
-      run: testing/setup_envs.sh purge-packages
+      run: |
+        source $CONDA/etc/profile.d/conda.sh
+        conda config --add channels defaults
+        conda config --add channels conda-forge
```

Comment:
Are both really needed here?

Reply:
This is also possible in a single command `conda config --append channels conda-forge --append channels defaults --append channels example`, it will warn about either moving the channel to the top (with --add) or to the bottom (with --append) if it is already in the list.

Reply:
Probably not, I will keep conda-forge.